lib/resources/majors.tsx

```diff
@@ -1822,153 +2010,173 @@ const bsCS_Old: Major = {
-          criteria: 'AS.110.108[C]^OR^AS.110.109[C]',
+            '<b>Calculus I</b> <br /> AS.110.108 Calculus I (Physical Sciences & Engineering)',
+          required_credits: 4,
+          criteria: 'AS.110.108[C]',
         },
         {
           description:
-            '<b>Math Elective:</b><p>This must be 200-level or above, chosen from Mathematics (AS.110.xxx) or Applied Math and Statistics (EN.553.xxx)</p>',
+            '<b>Calculus II</b> <br /> AS.110.109 Calculus II (Physical Sciences & Engineering)',
           required_credits: 4,
-          criteria: 'AS.110.[C]^OR^EN.553.[C]',
+          criteria: 'AS.110.109[C]',
         },
-      ],
-    },
-    {
-      name: 'Probability and Statistics Requirements',
-      required_credits: 4,
-      min_credits_per_course: 3,
-      description:
-        '<p> The BS math courses must include coverage ' +
-        'of both probability and statistics, which can be satisfied in many ways, including ' +
-        'taking any of the 553.3xx combined Probability & Statistics courses. Probability and Statistics:</p><p>Two paths:</p>',
-      criteria: 'AS Mathematics[D]^OR^EN Applied Mathematics & Statistics[D]',
-      exception:
-        '(Probability & Statistics[N]^OR^Probability and Statistics[N]^OR^EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]^OR^EN.553.420[C]^OR^EN.553.430[C])',
-      pathing: true,
-      fine_requirements: [
         {
           description:
-            '<p>1. Any of the three courses below:</p><p>EN.553.211</p><p>EN.553.310</p><p>EN.553.311</p> ',
+            '<b>Discrete Mathematics</b> <br /> EN.553.171 Discrete Mathematics',
           required_credits: 4,
-          criteria:
-            'Probability & Statistics[N]^OR^Probability and Statistics[N]^OR^EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]',
+          criteria: 'EN.553.171[C]',
         },
         {
+          // TODO: How to account for prob/stats coverage requirement?: MATH ELECTIVES DISTRIBUTION WITH PATHING FOR PROB STATS
           description:
-            '<p>2. Both Intro to Probability and Intro to Statistics</p><p>EN.553.420</p><p>EN.553.430</p> ',
-          required_credits: 8,
-          criteria: 'EN.553.420[C]^OR^EN.553.430[C]',
+            '<b>Electives</b> <br /> At least 3 more courses must be taken at the 200 or above level, ' +
+            'and must include coverage of both Probability and Statistics.',
+          required_credits: 12,
+          criteria:
+            '(AS Mathematics[D]^OR^EN Applied Mathematics & Statistics[D])^AND^(200[L]^OR^Upper Level[L])',
         },
       ],
     },
     {
-      name: 'Science',
+      name: 'Basic Sciences', // TODO: How to account for language/linguistic courses?
       required_credits: 16,
       min_credits_per_course: 1,
       description:
         'At least two semesters of physics or two semesters of chemistry, with the associated laboratories, must be included.',
       criteria: 'N[A]',
```

Comment:
Because it is difficult to determine which physics/chemistry courses the students will take. For example, chem I and II is valid, but so is orgo I and II...I am also not sure which classes do end up counting as valid classes. So I just leave it up to the student to decide and expect them to make the right decision. Not the best...do you have suggestions for this?

Reply:
Hmm maybe use pathing for Introductory Chemistry I&II, Organic Chemistry I&II, and General Physics I&II? Really, I think those are _the_ classes CS students take.

Reply:
should be fine I think. I will also fix the CS major later.